scrapers/lethalpass.yml

```
@@ -0,0 +1,32 @@
+name: lethalpass
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lethalpass.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class="pdSRC"]//a[starts-with(@href,"/sites/")]
```

The title now is kinda `useless`....
```suggestion
        selector: //div[@class="title"]/h1/text()|//div[@class="pdSRC"]//a[starts-with(@href,"/dvds/")]
```
The suggested one adds a `/text()` to the one you used before so no `hd` is printed/matched in case of dvds and the movie title is used instead... 
Does the suggeted one work well for you (and is it enough or should we try to get the ep number also) ?